### PR TITLE
mods identifier <--> cocina mapping specs reworked

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made?

reduce ongoing costs of mods/cocina mappings

## How was this change tested?

changes to specs only

### CODE REVIEW

**Review by jcoyne — APPROVED** [review]

File: spec/services/cocina/from_fedora/descriptive/identifier_spec.rb [hunk 1]
```diff
@@ -19,34 +19,8 @@
...
-                "uri": 'http://id.loc.gov/vocabulary/identifiers/'
-              }
-            }
-          ]
-        }
-      ]
-    end
-  end
-
+  # MODS identifier mapping spec not in /mappings/mods/identifier_spec.rb
```
Comment (justinlittman) [inline spec/services/cocina/from_fedora/descriptive/identifier_spec.rb:22]: I'd suggest that these comments be omitted, as they will take another PR to remove later. I don't think comments are a good way to track this.

**Review by justinlittman — APPROVED** [review]
Approve pending consideration of comment.

File: spec/services/cocina/from_fedora/descriptive/identifier_spec.rb (hunk 1 above)
Comment (ndushay) [reply spec/services/cocina/from_fedora/descriptive/identifier_spec.rb:22]: How do you envision we'll deal w these?  Arcadia go through all the remaining from_ and to_ fedora specs and pull what she wants into the new style mapping ones?   And then devs go through what remains?

I'm open to not using the comments and I'm open to removing the existing ones I've put in this PR and merged ones ... but I'd like to work out a plan w Arcadia.

Comment (ndushay) [reply spec/services/cocina/from_fedora/descriptive/identifier_spec.rb]: Per slack discussion, I am mollified.  I have removed the offending comments from this PR and will remove the others I have added elsewhere in a separate PR.

**Pull request merged**